get_github_coderabbit_review/coderabbit_fetcher/async_processing/async_task_manager.py

```
@@ -0,0 +1,594 @@
+                logger.info(f"Executing wave {wave_number + 1} with {len(task_ids)} tasks")
+                await self._execute_task_wave(task_ids)
+
+            execution_time = time.time() - start_time
+            self.stats["total_execution_time"] = execution_time
+
+            # Generate summary
+            summary = self._generate_execution_summary()
+            summary["execution_time_seconds"] = execution_time
+
+            logger.info(f"All tasks completed in {execution_time:.2f}s")
+            return summary
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)